packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -147,6 +152,60 @@ export type BaseInputProps = InputLabelProps &
      * Displays the character counter under the input field
      */
     maxCharacters?: number;
+    /**
+     * alignement of the text inside input field
+     */
+    textAlign?: 'left' | 'center' | 'right';
+    /**
+     * If true, focuses the input field on load
+     *
+     * **Note:**
+     * Automatically focusing a form control can confuse visually-impaired people using screen-reading technology and people with cognitive impairments.
+     * When autofocus is assigned, screen-readers "teleport" their user to the form control without warning them beforehand.
+     */
+    autoFocus?: boolean;
+    /**
+     * determines what return key to show on keyboard on mobile devices/virtual keyboard
```

Comment:
```suggestion
     * determines what return key to show on the keyboard of mobile devices/virtual keyboard
```

Reply:
fixed in b8929d24b4e7a0d218bba7e599803658e69dc4a8